### PR TITLE
Fix copying of debian fixtures

### PR DESCRIPTION
**Pull request by mdellweg**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,6 +19,7 @@ jobs:
         - docker build -t quay.io/pulp/pulp-fixtures .
         - docker run -d -p 8000:80 quay.io/pulp/pulp-fixtures
         - curl --fail localhost:8000/fixtures/file/PULP_MANIFEST
+        - curl --fail localhost:8000/fixtures/debian/dists/ragnarok/Release
```
Comment (quba42) [inline .travis.yml]: Adds a test to ensure there is at least some content from each build container present. ACK!

File: debian/gen-fixtures.sh [hunk 2]
```diff
@@ -51,5 +51,6 @@ reprepro -C jotunheimr includedeb ragnarok jotunheimr/*.deb
 
 rm dists/ragnarok/jotunheimr/binary-armeb/Packages
 
-cp -r --no-preserve=mode --reflink=auto "${TMPDIR}"/dists "${OUTPUTDIR}"
-cp -r --no-preserve=mode --reflink=auto "${TMPDIR}"/pool "${OUTPUTDIR}"
+mkdir -p "${OUTPUTDIR}"
```
Comment (quba42) [inline debian/gen-fixtures.sh]: Ensures the two following commands actually both work in the same way. ACK!

File: debian/gen-fixtures.sh [hunk 3]
```diff
@@ -51,5 +51,6 @@ reprepro -C jotunheimr includedeb ragnarok jotunheimr/*.deb
...
+cp -r --no-preserve=mode --reflink=auto "${TMPDIR}"/dists -t "${OUTPUTDIR}"
+cp -r --no-preserve=mode --reflink=auto "${TMPDIR}"/pool -t "${OUTPUTDIR}"
```
Comment (quba42) [inline debian/gen-fixtures.sh]: Do we need both the `-t` and the `mkdir -p`?

Comment (mdellweg) [reply debian/gen-fixtures.sh]: No, the `-t` would fail if the directory didn't exist. So it is there as a safeguard.

**Pull request merged**